src/applications/personalization/profile/components/Profile.jsx

```diff
@@ -294,20 +294,25 @@ const mapStateToProps = state => {
   const showLoader =
     !hasLoadedAllData || (!isLOA3 && !isLOA1 && currentlyLoggedIn);
 
+  const shouldShowDirectDeposit = () => {
+    // if they are explicitly blocked from DD4CNP, do not show it
+    if (isCNPDirectDepositBlocked) return false;
+    return (
+      (isLOA3 && !is2faEnabled) || // we _want_ to show the DD section to non-2FA users
+      isCNPDirectDepositSetUp ||
+      isEligibleToSetUpCNP ||
```

Comment:
Do we no longer need to check for `shouldFetchEDUDirectDepositInformation`? (`!!showDirectDepositV2(state) && is2faEnabled`)

Reply:
Good catch. Yeah, I see no reason why we needed to fetch that in the first place.